AppAuth-JS-Ext-Logout/built/end_session_redirect_based_handler.js

```diff
@@ -1,16 +1,20 @@
 "use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file